File: Csharp/MPP/model2/Organiser.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace model2
{
    [Serializable]
    public class Organiser : Identifiable<int>
    {
        public Organiser()
        {
        }

        public Organiser(int id)
        {
            Id = id;
        }

        public Organiser(string name, string password) : this(name)
        {
            Password = password;
        }

        public Organiser(int id, string name, string password) : this(id, name)
        {
            Password = password;
        }

        public virtual int Id { get; set; }

        public virtual string Name
        {
            get;
            set;
        }

        public virtual string Password
        {
            get;
            set;
        }

        public Organiser(int id, string name)
        {
            Id = id;
            Name = name;
        }

        public Organiser(string name)
        {
            Name = name;
        }

        public override string ToString()
        {
            return Id + " " + Name;
        }
    }
}

File: JavaSockets/BaskModel/src/main/java/bask/model/Organiser.java
package bask.model;

import org.hibernate.annotations.GenericGenerator;

import java.io.Serializable;
import javax.persistence.Entity;
import javax.persistence.GeneratedValue;
import javax.persistence.Table;


//@Entity
//@Table( name = "Organiser")
public class Organiser implements Identifiable<Integer>, Serializable {
    private Integer id;
    private String name;
    private String password;

    public Organiser(Integer id, String name) {
        this.id = id;
        this.name = name;
    }

    public Organiser(String name, String password) {
        this.name = name;
        this.password = password;
    }

    public Organiser() {
    }

    public Organiser(Integer id){
        this.id = id;
    }

    public Organiser(String name) {
        this.name = name;
    }

    public Organiser(int id, String name, String password) {
        this.id = id;
        this.name = name;
        this.password = password;
    }

    @Override
//    @javax.persistence.Id
//    @GeneratedValue(generator="increment")
//    @GenericGenerator(name="increment", strategy = "increment")
    public Integer getId() {
        return id;
    }

    @Override
    public void setId(Integer id) {
        this.id = id;
    }

    public String getName() {
        return name;
    }

    public String getPassword() {
        return password;
    }

    public void setPassword(String password) {
        this.password = password;
    }

    public void setName(String name) {
        this.name = name;
    }

    @Override
    public String toString() {
        return id + " " + name + " " + password;
    }
}
